scripts/plugins/logging.py

```diff
@@ -1,11 +1,14 @@
 import logging
+import re
 import sys
+from munch import DefaultMunch
 
 from .settings import (
     BITOPS_logging_level,
     BITOPS_logging_color,
     BITOPS_logging_filename,
     BITOPS_logging_path,
+    bitops_build_configuration,
```

Comment:
Please extract the masks in the settings.py file instead of sending the entire configuration.

Example; 
```
BITOPS_logging_path = (
    bitops_build_configuration.bitops.logging.path
    if bitops_build_configuration.bitops.logging.path is not None
    else "/var/log/bitops"
)

BITOPS_logging_masks = (
    bitops_build_configuration.bitops.logging.masks
    if bitops_build_configuration.bitops.logging.masks is not None
    else None
)
```